src/app/components/top/reglas/visualizacionReglas.component.ts

```diff
@@ -130,6 +137,42 @@ export class VisualizacionReglasComponent implements OnInit {
             });
         }
     }
+
+    public seleccionarConcepto(concepto) {
+        concepto['esSolicitud'] = true;
+
+        const params: any = {
+            estados: [
+                'auditoria',
+                'pendiente',
+                'ejecucion'
+            ],
+            idPaciente: this.ejecucionService.paciente.id,
+            prestacionDestino: concepto.conceptId
+        };
+        this.servicioPrestacion.getSolicitudes(params).subscribe(resultado => {
+            if (resultado.length) {
+                this.plex.confirm(`El paciente ya tiene una solicitud en curso para ${concepto.term}. ¿Desea continuar?`, 'Paciente con solicitud en curso').then(confirmar => {
+                    if (confirmar) {
+                        this.agregarConcepto(concepto);
+                    }
+                });
+                this.plex.toast('danger', `El paciente ya tiene una solicitud en curso para ${concepto.term}`);
+            } else {
+                this.agregarConcepto(concepto);
+            }
+        });
+
+    }
+
+    private agregarConcepto(concepto) {
+        this.ejecucionService.agregarConcepto({
+            term: concepto.term,
+            fsn: concepto.fsn,
+            conceptId: concepto.conceptId,
+            semanticTag: concepto.semanticTag
+        }, concepto.esSolicitud);
```

Comment:
Acá no se puede directamente enviar el concepto, en lugar de traspasar los atributos a un objeto con los mismos keys?
```suggestion
        this.ejecucionService.agregarConcepto(concepto, concepto.esSolicitud);
```